protos/commands.proto

```diff
@@ -17,49 +17,102 @@ enum Type {
 	ARTIST_COUNT = 8;
 	FOLDER_COUNT = 9;
 	SELECT_QUERY = 10;
-	CONTENT_QUERY = 11;
 	WHEN_DEVICE = 12;
```

Comment:
Reserved seems to mean something else. Did not work in an enum. Think it is to preserve sizing / soacing in a message.